testdata/p4_16_samples_outputs/issue561-2-bmv2-frontend.p4

```diff
@@ -66,8 +66,10 @@ control ingress(inout headers hdr, inout metadata meta, inout standard_metadata_
     apply {
         debug_hdr_0.apply();
         if (hdr.u.short.isValid()) {
+            hdr.u.short.data = 16w0xffff;
```

Comment:
What's happening here?

Reply:
This is an instance of the same change as below: the setInvalid call that follows this statement is no longer considered as overwriting all fields, so this assignment is no longer considered dead.